game_patch/rf/player/player.h

```diff
@@ -173,4 +173,6 @@ namespace rf
     static auto& player_do_frame = addr_as_ref<void(Player*)>(0x004A2700);
     static auto& player_make_weapon_current_selection = addr_as_ref<void(Player *player, int weapon_type)>(0x004A4980);
     static auto& player_start_death_fade = addr_as_ref<void(Player *pp, float time_sec, void (*callback)(Player *))>(0x004A73E0);
+    static auto& game_get_gore_level = addr_as_ref<int()>(0x00436A20);
+    static auto& game_set_gore_level = addr_as_ref<int(int gore_setting)>(0x00436A10);
```

Comment:
This should return `void`, like a standard setter.

Explanation in case you are wondering:
 Declaring it as void doesn't break anything - just makes EAX register ignored on return.
I know some types in IDA database are not 100% valid. It's hard to get all of them right without spending hundreds of years on it :) But when I copy definitions to Dash I want them to be correct and close to author intentions. 
IDA generally prefers to make all functions return something (because there's no such thing as `void` in machine code) even if it complicates it's de-compiled code. When I see that no-one uses the return value and my intuition says that function should not return anything I switch to `void`.

Reply:
Thank you for that explanation, it makes complete sense and you explained it very well. I will make that adjustment here and this will also help me understand such things going forward.